catalog/diff.go

```diff
@@ -11,10 +11,16 @@ const (
 )
 
 type Difference struct {
-	Entry                // Partially filled.
+	Entry                // Partially filled. Path is always set.
 	Type  DifferenceType `db:"diff_type"`
 }
 
+type diffResultRecord struct {
```

Comment:
The type is still private, other packages will have odd behavior while trying to use the iterator returned Value()